config/application.rb

```diff
@@ -19,7 +19,8 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    #config.time_zone = 'Central Time (US & Canada)'
+    ENV['TZ']='UTC'
```

Comment:
Actually, yes I definitely want to leave it where it is, because I run this site on a non-utc system.  If not set I couldn't do any type of "manual" testing with events as the times would be all screwed up.